<launch>
<include file="$(find sim2real_dynamics_sawyer)/launch/observations/manual_positioning.launch"/>
    <node name="reach_control" pkg="sim2real_dynamics_sawyer" type="reach_control.py" output="screen" >
    </node>
</launch>